libs/perun/dialogs/src/lib/applications-bulk-operation-dialog/applications-bulk-operation-dialog.component.scss

```diff
@@ -0,0 +1,12 @@
+.dialog-content {
+  padding-top: 0 !important;
+  max-height: 300px;
+  max-width: 95%;
+  overflow-y: auto;
+  overflow-x: auto;
+}
```

Comment:
I would most of this we already get by using the `mat-dialog-content`. You should set up the dialog so that all content is under one parent with `mat-dialog-content`.